src/main/webapp/resources/js/components/project-members/AddMemberButton.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { useContext, useEffect, useRef, useState } from "react";
+import {
+  Button,
+  Form,
+  Modal,
+  notification,
+  Radio,
+  Select,
+  Typography,
+} from "antd";
+import {
+  addMemberToProject,
+  getAvailableUsersForProject,
+} from "../../apis/projects/members";
+import { useDebounce, useResetFormOnCloseModal } from "../../hooks";
+import { SPACE_XS } from "../../styles/spacing";
+import { PagedTableContext } from "../ant.design/PagedTable";
+import { ProjectRolesContext } from "../../contexts/ProjectRolesContext";
+
+const { Option } = Select;
+const { Text } = Typography;
+
+/**
+ * React component that adds a button to open a modal to add a new member
+ * to a project.  User can search for a user but first name, last name or
```

Comment:
Change `but` to `by`

Reply:
Fixed in 5b4a37e24b443d99df6ef574ff99fe5368db3ebb

Reply:
Thanks